---
title: "Projects &amp; Interests"
permalink: interests/
meta: "Big Data Biology Lab Projects &amp; Interests"
---

## Small proteins of the global microbiome

Small proteins (exact definitions vary, but &lt; 100 amino acids is a common
threshold) have generally been ignored as standard bioinformatics approaches
fail when applied to these short sequences. Nonetheless, it has also become
clear that small proteins are abundant across all domains of life.

As an initial project in this domain, we have focused on antimicrobial peptides (AMPs)
and developed [macrel]({{ site.baseurl }}/software/macrel), a tool to mine them
from genomes and metagenomes. We are currently building up a catalog of AMPs in
the global microbiome and analysing how this mechanism is used to shape
microbial communities.

### Relevant papers &amp; Links

- **Célio Dias Santos-Junior**, **Shaojun Pan**, Xing-Ming Zhao, **Luis Pedro
   Coelho** [MACREL: antimicrobial peptide screening in genomes and
   metagenomes](https://peerj.com/articles/10555/), PeerJ 2020

## Antimicrobial resistome

Antimicrobial resistance (AMR), whereby bacteria become resistant to antibiotics,
is a major global public health problem. Our research interests are centered around antimicrobial resistome
 – bacterial genes encoding antibiotic resistance found just about everywhere, 
from the deep ocean to healthy human gut, and important for both environmental and clinical settings. 
We are [a part of the international EMBARK]({{ site.baseurl }}/projects/embark) project, which analyses AMR at a
global scale.

### Relevant papers &amp; Links

- [EMBARK Consortium main page](https://antimicrobialresistance.eu/)


## Similarities and contrasts in the microbial ecology of different habitats

We are interested in looking at different habitats and comparing their
microbial ecology.

What are the common features of different microbiota (for example, the gut of
different mammals)? Are strains biome-specific?

### Relevant papers

- **Luis Pedro Coelho**, Jens Kultima, Paul Costea, Coralie Fournier,
  Yuanlong Pan, Gail Czarnecki-Maulden, Matthew Hayward, Sofia
  K. Forslund, Patrick Descombes, Janet Jackson, Qinghong Li, and Peer
  Bork [Similarity of the dog and human gut microbiomes in gene content
  and response to diet](https://doi.org/10.1186/s40168-018-0450-3)
- Paul I Costea, **Luis Pedro Coelho**, Shinichi Sunagawa, Robin Munch, Jaime
  Huerta‐Cepas, Kristoffer Forslund, Falk Hildebrand, Almagul Kushugulova,
  Georg Zeller, Peer Bork [Subspecies in the global human gut
  microbiome](http://doi.org/10.15252/msb.20177589)
- Shinichi Sunagawa\*, **Luis Pedro Coelho**\*, Samuel Chaffron\*, [...]
  Matthew B. Sullivan, Jean Weissenbach, Patrick Wincker, Eric Karsenti,
  Jeroen Raes, Silvia G. Acinas, Peer Bork, [Structure and function of
  the global ocean microbiome](http://doi.org/10.1126/science.1261359)


## Developing tools for very large-scale metagenomic analysis

We are interested both in developing new algorithms with applications to
large-scale metagenomics and in exploring tools that enable reproducible
science.

In particular, we are interested in the potential of designing domain specific
languages to concisely specify data handling pipelines in a way that is (1)
intuitive, (2) enables defensive programming, (3) computational efficient, (4)
extensible.

### Relevant papers &amp; Links

- **Shaojun Pan**, **Chengkai Zhu**, Xing-Ming Zhao⁺, and **Luis Pedro
  Coelho**⁺ [SemiBin: Incorporating information from reference genomes with
  semi-supervised deep learning leads to better metagenome-assembled genomes
  (MAGs)](https://doi.org/10.1101/2021.08.16.456517) BioRxiv preprint (2021).
- **Luis Pedro Coelho**, Renato Alves, Paulo Monteiro, Jaime Huerta-Cepas, Ana
  Teresa Freitas, Peer Bork [NG-meta-profiler: fast processing of metagenomes
  using NGLess, a domain-specific
  language](https://doi.org/10.1186/s40168-019-0684-8) in Microbiome (2019).
  See also the [NGLess webpage](https://ngless.embl.de)
- **Luis Pedro Coelho** [Jug: Software for parallel reproducible computation in
  Python](http://doi.org/10.5334/jors.161) in Journal of Open Research Software.
  5(1), p.30, (2017).

## Exploring large-scale metagenomics datasets

We are interested in exploiting very large scale metagenomic datasets for basic
ecological questions.

What can we learn about microbes and their interaction patterns? Can we use
model metatranscriptomes at a community-wide scale?

### Relevant papers &amp; Links

## Pushing the boundaries on bioimage informatics

We want to push the boundaries on bioimage informatics. This can mean _work on
larger datasets_ or _build detailed models from images_ or _link images and
metagenomics_ or something else you can think of.

### Relevant publications

- Sebastien Colin\*, **Luis Pedro Coelho**\*, Shinichi Sunagawa, Chris Bowler,
  Eric Karsenti, Peer Bork, Rainer Pepperkok, Colomban de Vargas, [Quantitative
  3D-Imaging of Environmental Microbial Eukaryotes using Automated Fluorescence
  Microscopy](https://doi.org/10.7554/eLife.26066.001) in eLife (2017).
- **Luis Pedro Coelho**, Catarina Pato, Ana Friães, Ariane Neumann, Maren von
  Köckritz-Blickwede, Mário Ramirez, João André Carriço, [Automatic
  determination of NET (neutrophil extracellular traps) coverage in fluorescent
  microscopy images](https://doi.org/10.1093/bioinformatics/btv156). in
  Bioinformatics (2015)

## Possible collaborations

We are interested in working with clinical groups on issues of relevance to
human health or with groups with access to environmental samples.

Please [get in touch](mailto:luispedro@big-data-biology.org).

