---
title: "Cryptic antimicrobial peptides (AMPs) in the human gut: a potential case study"
permalink: blog/2020-04-10-cryptic/
---

_Amy Houseman, Célio Dias Santos-Junior, Luis Pedro Coelho_

## Introduction I: what are AMPs?

Antimicrobial Peptides are short protein molecules, usually 10-100
residues in length, that interfere with microbial cell function,
particularly on membrane disruption. AMPs are widespread among the three
domains of life. Due to the reduced size of AMPs, their identification
usually relies on the ability to distinguish between their sequence
composition, structure and physico-chemical features (Meher *et al*.,
2017). After their predominant discovery in the 1980s, AMPs have shown
to have a broad spectrum of activity, such as some AMPs are
anti-bacterial (Jiravanichpaisal *et al*., 2007, Li *et al*., 2017),
anti-fungal (Gupta and Srivastava., 2014, Do *et al*., 2014) and
anti-cancer (Hilchie *et al*., 2011).

AMPs vary both in sequence and abundance among the three domains of
life. For example, the Antimicrobial Peptide Database (AMPdb) (Wang *et
al*., 2016; also [*http://aps.unmc.edu/AP/*](http://aps.unmc.edu/AP/))
which contains known AMP sequences is composed by only 12% of reported
sequences belonging to prokaryotes, while its majority (88%) is
eukaryotic. AMPs are a diverse group of molecules, commonly cationic.
This means they have a positive charge, and usually organised faces with
hydrophobic and hydrophilic regions. These regions enable AMPs to
interact with microbial membranes (Cunha *et al*., 2017). The mechanism
of action of prokaryotic AMPs can be divided into two main categories;
membrane-targeted disruption or non-membrane targeted inactivation. The
non-membrane mechanism usually refers to those AMPs that interact with
microbial cell walls or intracellular targets, being able or not to
cause cell lysis.

Interestingly, some AMPs have also shown to be effective against
antibiotic-resistant bacteria (Lázár *et al*., 2018). With the prominent
antimicrobial-resistance (AMR) crisis (Vladislav *et al*., 2019), AMPs
could represent an alternative to antibiotics. With a variety of sectors
exploiting AMP properties, the need to understand their evolution and
adaptive capabilities has never been higher.

## Introduction II: what are cryptic AMPs?

There are known AMPs which originate from the breakup of larger
proteins. There are a variety of ways that cryptic AMPs can become
activated, one well-documented way is with the use of naturally
occurring proteases. These proteases are able to cleave the parent
protein, releasing the AMP.

Lactoferricin is a cryptic AMP which works in this specific way, being
released from lactoferrin, its parent protein. Lactoferrin is typically
found in human and cow milk, although it has been found in saliva and
other bodily fluids. Lactoferrin is an iron-binding glycoprotein, and
has been recognised as being an important aspect in both systemic and
gastrointestinal immune development. Lactoferricin, a cryptic peptide,
was discovered in 1992 through the pepsin cleavage of lactoferrin in the
gut of humans (Bellamy *et al*. 1992). A paper by Gifford *et al*.,
(2005) highlighted lactoferricin’s potential bioactivity, including its
ability to be anti-bacterial, antiviral and antitumor.

Cryptic AMPs can also be formed by the activation of protein precursors.
This activation can be triggered by a variety of situations, namely
external stress conditions. An example of this in plants is that a
phytohormone treatment induced cryptic AMPs in moss (Fesenko *et al*.,
2019).

Searching for AMPs in human gut metagenome data (Heintz-Buschart *et
al*., 2016), led us to find a potential cryptic peptide formed from
mutations in a protein sequence.

## HG4: Probably an AMP

Another method by which cryptic AMPs can naturally form is *de novo* (as
they have reduced length) or by the direct disruption of the genes
coding for larger proteins. Using 184 human guts metagenomes data from
Heintz-Buschart *et al*., (2016), we assembled the reads into contigs
using MegaHit v.1 (Li *et al*., 2016; also
https://github.com/JGI-Bioinformatics/megahit). We then performed a
screening with Macrel v 0.4 (Santos-Junior *et al.*, 2019; also
[github.com/BigDataBiology/macrel).](https://github.com/BigDataBiology/macrel)
Macrel v 0.4 consists of 2 classifiers using 22 features, that predict
AMP sequences within metagenomic sequences. Macrel uses a novel set
which mixes well-known variables with some innovation, such as the
distribution of residues by solvent accessibility and free energy
transfer to water from its lipophilic phase, to detect AMPs. These
features increase its precision and reliability, even by using a small
number of them.

[Within this dataset, we recovered 102 metagenomes containing putative
AMPs.](https://github.com/BigDataBiology/macrel) One of them, HG4, was
found in three metagenomes from the same subject. Using homology, HG4
was identified within a sequence from *Prevotella melaninogenica.*

We aligned the contig containing HG4 sequence with the *Prevotella
melaninogenica* genome, and discovered that the portion containing the
HG4 AMP was disrupted. Specifically, there was a deletion of a
nucleotide upstream of the HG4 peptide start codon, this deletion
disrupted the amino terminal and created a ribosome binding site with a
11 bp spacer, shown in figure 1. Further, downstream a point mutation
occurred, generating a stop codon. This complete small ORF would be
transcribed by the original gene promoter and its structure seems prone
to be translated.

![](/media/2020-04-10-cryptic/image1.png)

**Figure 1.** Diagram of the mutations that result in the formation of the AMP
HG4. A point deletion occurs (Vdel) led to the creation of a ribosome binding
site. Further, downstream, at the 25th residue, a second point mutation (G
-&gt; A) results in a stop codon.

A few further tests served to strengthen our prediction that HG4 has
antimicrobial properties. The first being a hydrophobicity scan of the
reconstituted protein sequence containing HG4 with the Eisenberg scale.
Hydrophobicity usually refers to the percent of hydrophobic residues in
a peptide sequence and is essential for AMPs to partition into the
membrane bilayer (Kumar *et al*., 2018). As can be seen in Figure 2**,**
the sequence containing the AMP (first 100 residues) is a transition
from hydrophobic to hydrophilic regions. Therefore, the typical
amphiphilic profile indicates the HG4 as likely to be an AMP.

![](/media/2020-04-10-cryptic/image2.png)

**Figure 2.** Hydrophobicity profile of the reconstituted protein
containing HG4. With the majority of points having a score above 0, we
can say that this sequence is amphiphilic, and likely to be that of an
AMP.

AMPs usually have alternating sections of polar and hydrophobic
residues, which is essential for them to facilitate membrane movement
(Dennison *et al*., 2005). HG4 has polar residues mostly arranged along
one side of the helical wheel presented in Figure 3**.** For AMPs to
interact with their target microbes, they require an electrostatic
charge. An electrostatic attraction exists between the negative charge
of a bacterial membrane, and the positively charged AMP. The positive
charge in the AMP is caused by specific residues in the primary
sequence, and is responsible for the formation of pores. These pores
lead to cytoplasm leakage in the target cell, causing the cell’s death (Lei *et
al*., 2019).

![](/media/2020-04-10-cryptic/image3.png)

**Figure 3**. Amino Acid wheel of HG4, showing polarity on the left hand
side, represented with green circles

We tried to find any ortholog sequences for HG4 using the EggNog
database (Huerta-Cepas *et al*., 2019; also
[*http://eggnog5.embl.de*](http://eggnog5.embl.de/))
and chose the cluster with the best score and lowest e-value. We scanned
the orthologs of the reconstituted protein containing HG4 regarding to
the AMP probability, by slicing a sequence into k-mers and predicting
with Macrel with the option `--keep-negatives` this showed us how likely the
fragments are to be an AMP at each position after removing signal
peptides. The scans of *Prevotella sp*. did look similar to the pattern
expressed by our reconstituted protein containing HG4 original AMP
sequence.

One hypothesis would be that *P. melaninogenica* produces the whole
protein, and through the human digestive tract it is degraded into the
HG4 peptide, what would argue in the sense that HG4 was a cryptic
peptide. Figure 4 shows the protease cleavage sites predicted with
Peptide Cutter (Artimo *et al.*, 2012; and
https://web.expasy.org/peptide\_cutter/) from the reconstituted protein
containing HG4 . This shows that general proteases from the human
intestine would be able to cleave the complete protein, releasing HG4 by
proteolysis. At this point, although speculative, it is tempting to
think about the fate of potential cryptic AMPs alongside evolution.

The reconstituted protein is from a specific family of oxidoreductase.
This enzyme has a signal peptide which was detected using SignalP4.0
(Nielson., 2017; and
[*http://www.cbs.dtu.dk/services/SignalP/*](http://www.cbs.dtu.dk/services/SignalP/)).
Signal peptides present in prokaryotic proteins are likely used for the
secretion of said protein. Our reconstituted protein has a signal
peptide, hence is probably secreted. Oxidoreductases are dependent on
oxygen, which is depleted in human guts. So, a disruption in this gene
would be less harmful for a microbe in the guts, such as *Prevotella
sp.* Furthermore, its secretion also argues in the sense that the
protein originally would be available for proteolysis.

![](/media/2020-04-10-cryptic/image4.png)

**Figure 4.** This shows the available
protease sites in the reconstituted protein containing the peptide HG4
(red dashed box) for proteases from the human intestinal environment*.*
Trypsin and Chymotrypsin-high specificity are both able to cleave the
protein in a position that are beneficial in the extraction of HG4.

HG4 also had the longest predicted half-life among all the possible
peptides (generated in silico) formed from the reconstituted protein,
and consequently the lowest decay. Half-life was calculated in an
intestinal-like environment because proteases present in the intestine
have the ability to form HG4. Using the HLP server (Sharma *et al*.,
2014; also
[*http://crdd.osdd.net/raghava/hlp/index.html*](http://crdd.osdd.net/raghava/hlp/index.html)),
we predicted that this peptide would be stable in the human gut for at
least 3 seconds, relatively a long half-life (Figure 5). Half-life is
defined as the time needed for the peptide amount to be halved. Figure 6
shows that HG4, once generated, is stable in the human gut for at least
3 seconds. Building on from previous speculation, if the reconstituted
protein is present in the human gut for a prolonged period of time and
proteases release HG4 at specific sites, this AMP would have enough time
to interact with its target due to its stability.

![](/media/2020-04-10-cryptic/image5.png)

**Figure 5.** Half-life (seconds) plotted against peptide positions of
the reconstituted HG4 gene sliced in 25-mer. The position of the
antimicrobial peptide, HG4, shows HG4 has an average half-life of 3
seconds. The rest of the peptidyl positions have a faster half-life than
the position of HG4, therefore being degraded faster in an
intestinal-like environment than HG4.

![](/media/2020-04-10-cryptic/image6.png)

**Figure 6.** Decay rate of each peptidyl position of the reconstituted
protein. HG4 is in the position labelled with a red circle and has the
lowest decay in comparison to all other peptide positions. This means it
remains in its original state for a period of time longer than the other
positions on the peptide.

We also searched for the HG4 sequence in the Global Microbial Gene
Catalogue version 1 ([*http://gmgc.embl.de/*](http://gmgc.embl.de/)) and
had a hit from two saliva samples from the same donor that contained the
HG4 gene. These hits, also predicted to be from *Prevotella*
*melaninogenica*, which suggests also that due to this species' niche,
known to cause tooth cavities, that human proteases could be working on
the mouth and guts to generate HG4 AMP.

## What is the target of HG4?

The prediction tool available in DBAASP database (Pirtskhalava *et al*.,
2016 and; [*https://dbaasp.org/*](https://dbaasp.org/)) predicts HG4
activity against *Klebsiella pneumoniae* (0.85 PPV)*.* With this
information, we searched the Global Microbial Gene Catalogue sample data
to see if there were any correlation between the abundance of genes
coding the cryptic HG4 and *Klebsiella pneumoniae*.

The Spearman’s correlation is not statistically significant
(correlation=0.14, p value=0.09) which is what we expected. However,
Fisher’s exact test on presence/absence (p= 1.155e-06) between HG4 and
*Klebsiella pneumoniae is* statistically significant. This suggests that
there is a non-random association between the AMP and its potential
target.

## Insights and valuable ideas from HG4**

![](/media/2020-04-10-cryptic/image7.png)

Here, we present the potential case of a cryptic AMP, HG4, from human
metagenome samples. HG4 was discovered using Macrel, an AMP screener. We have
identified at least two ways that this cryptic peptide may be isolated and
activated within a protein from *Prevotella melaninogenica*, these being
mutation-induced and through the use of human proteases. Diagram 1 shows the
basis by which these processes could occur. Although AMPs are widespread among
the three domains of life, there has been little documentation about the origin
of prokaryotic AMPs, and the current results could give insights about the
evolving processes behind them. We therefore show a potential mechanism for how
prokaryotic cryptic AMPs could be derived into formal genes from mutations in
bacterial proteins.

**Diagram 1**. Purple boxes represent the process by which proteases are
able to cleave HG4 from a protein found in *Prevotella melaninogenica*.
Green boxes represent the process by which two mutations occur within
the protein sequence from *Prevotella melaninogenica*, leading to the
formation of HG4.

The existence of cryptic AMPs within *Prevotella melaninogenica* would
suggest that prokaryotes harvest a collection of hidden antimicrobial
sequences that are currently unexplored. Using AMP screening tools, it
should be possible to source AMP origins and replicate these in
vivo/vitro for clinical use. Although speculative, the information about
HG4 can serve to lead in future to solid conclusions. Also, we have
identified a way by which certain mutations can create AMP genes. If
more AMPs created from mutations are found, we may be able to predict
the likelihood of certain mutations creating AMPs. With this knowledge,
we could exploit known prokaryotic proteins in ways that have not been
done before.

For now, we have an isolated case study, but it is interesting to ask
(1) whether these mutations are positively selected for and (2) how
often do these events happen across the microbial world. This is one of
the avenues we keep pursuing. Hopefully, we will be able to give you an
update in the not so distant future.

## References

Artimo, P., Jonnalagedda, M., Arnold, K., Baratin, D., Csardi, G., De
Castro, E., Duvaud, S., Flegel, V., Fortier, A., Gasteiger, E. and
Grosdidier, A., 2012. ExPASy: SIB bioinformatics resource portal.
*Nucleic acids research*, 40(W1), pp.W597-W603.

Bellamy, W., Takase, M., Yamauchi, K., Wakabayashi, H., Kawase, K. and
Tomita, M., 1992. Identification of the bactericidal domain of
lactoferrin. *Biochimica et Biophysica Acta (BBA)-Protein Structure and
Molecular Enzymology*, 1121(1-2), pp.130-136.

Cunha, N.B., Cobacho, N.B., Viana, J.F., Lima, L.A., Sampaio, K.B.,
Dohms, S.S., Ferreira, A.C., de la Fuente-Núñez, C., Costa, F.F.,
Franco, O.L. and Dias, S.C., 2017. The next generation of antimicrobial
peptides (AMPs) as molecular therapeutic tools for the treatment of
diseases with social and economic impacts. *Drug discovery today*,
22(2), pp.234-248.

Dennison, S.R., Wallace, J., Harris, F. and Phoenix, D.A., 2005.
Amphiphilic α-helical antimicrobial peptides and their
structure/function relationships. *Protein and peptide letters*, 12(1),
pp.31-39.

Dias Santos-Junior, C., Pan, S.,, Zhao, XM., Coelho,LP. MACREL:
antimicrobial peptide screening in genomes and metagenomes. *bioRxiv*,
2019.12.17.880385; doi: https://doi.org/10.1101/2019.12.17.880385

Do, N., Weindl, G., Grohmann, L., Salwiczek, M., Koksch, B., Korting,
H.C. and Schäfer‐Korting, M., 2014. Cationic membrane‐active
peptides–anticancer and antifungal activity as well as penetration into
human skin. *Experimental dermatology*, 23(5), pp.326-331.

Fesenko, I., Azarkina, R., Kirov, I., Kniazev, A., Filippova, A.,
Grafskaia, E., Lazarev, V., Zgoda, V., Butenko, I., Bukato, O. and
Lyapina, I., 2019. Phytohormone treatment induces generation of cryptic
peptides with antimicrobial activity in the Moss Physcomitrella patens.
*BMC plant biology,* 19(1), pp.1-16.

Gifford, J.L., Hunter, H.N. and Vogel, H.J., 2005. *Lactoferricin.
Cellular and molecular life sciences*, 62(22), p.2588.

Gupta, R. and Srivastava, S., 2014. Antifungal effect of antimicrobial
peptides (AMPs LR14) derived from Lactobacillus plantarum strain LR/14
and their applications in prevention of grain spoilage. *Food
microbiology*, 42, pp.1-7.

Heintz-Buschart, A., May, P., Laczny, C.C., Lebrun, L.A., Bellora, C.,
Krishna, A., Wampach, L., Schneider, J.G., Hogan, A., De Beaufort, C.
and Wilmes, P., 2016. Integrated multi-omics of the human gut microbiome
in a case study of familial type 1 diabetes. *Nature microbiology,*
2(1), pp.1-13.

Hilchie, A.L., Doucette, C.D., Pinto, D.M., Patrzykat, A., Douglas, S.
and Hoskin, D.W., 2011. Pleurocidin-family cationic antimicrobial
peptides are cytolytic for breast carcinoma cells and prevent growth of
tumor xenografts. *Breast cancer research*, 13(5), p.R102.

Huerta-Cepas, J., Szklarczyk, D., Heller, D., Hernández-Plaza, A.,
Forslund, S.K., Cook, H., Mende, D.R., Letunic, I., Rattei, T., Jensen,
L.J. and von Mering, C., 2019. eggNOG 5.0: a hierarchical, functionally
and phylogenetically annotated orthology resource based on 5090
organisms and 2502 viruses. *Nucleic acids research*, 47(D1),
pp.D309-D314.

Jiravanichpaisal, P., Lee, S.Y., Kim, Y.A., Andrén, T. and Söderhäll,
I., 2007. Antibacterial peptides in hemocytes and hematopoietic tissue
from freshwater crayfish Pacifastacus leniusculus: characterization and
expression pattern. *Developmental & Comparative Immunology*, 31(5),
pp.441-455.

Kumar, P., Kizhakkedathu, J.N. and Straus, S.K., 2018. Antimicrobial
peptides: Diversity, mechanism of action and strategies to improve the
activity and biocompatibility in vivo. *Biomolecules*, 8(1), p.4.

Lázár, V., Martins, A., Spohn, R., Daruka, L., Grézal, G., Fekete, G.,
Számel, M., Jangir, P.K., Kintses, B., Csörgő, B. and Nyerges, Á., 2018.
Antibiotic-resistant bacteria show widespread collateral sensitivity to
antimicrobial peptides. *Nature microbiology*, 3(6), pp.718-731.

Lei, J., Sun, L., Huang, S., Zhu, C., Li, P., He, J., Mackey, V., Coy,
D.H. and He, Q., 2019. The antimicrobial peptides and their potential
clinical applications. *American journal of translational research*,
11(7), p.3919.

Li, N.N., Li, J.Z., Liu, P., Pranantyo, D., Luo, L., Chen, J.C., Kang,
E.T., Hu, X.F., Li, C.M. and Xu, L.Q., 2017. An antimicrobial peptide
with an aggregation-induced emission (AIE) luminogen for studying
bacterial membrane interactions and antibacterial actions. *Chemical
Communications*, 53(23), pp.3315-3318.

Meher, P.K., Sahu, T.K., Saini, V. and Rao, A.R., 2017. Predicting
antimicrobial peptides with improved accuracy by incorporating the
compositional, physico-chemical and structural features into Chou’s
general PseAAC. *Scientific reports*, 7(1), pp.1-12.

Nielsen, H., 2017. Predicting secretory proteins with SignalP. In
Protein function prediction (pp. 59-73). *Humana Press*, New York, NY.

Pirtskhalava M, Gabrielian A, Cruz P, Griggs HL, Squires RB, Hurt DE,
Grigolava M, Chubinidze M, Gogoladze G, Vishnepolsky B, Alekseev V,
Rosenthal A, and Tartakovsky M. DBAASP v.2: an Enhanced Database of
Structure and Antimicrobial/Cytotoxic Activity of Natural and Synthetic
Peptides. *Nucl. Acids Res*., 2016, 44 (D1), D1104-D1112.

Sharma A., Singla D., Rashid M. and Raghava G.P.S (2014) Designing of
peptides with desired half-life in intestine-like environment. *BMC
Bioinformatics* 2014, 15:282

Vladislav, M., Chernov, Olga, A., Chernova, Alexey, A., Mouzykantov,
Leonid, L., Lopukhov, Rustam, I., Aminov. (2019) Omics of antimicrobials
and antimicrobial resistance. *Expert Opinion on Drug Discovery* 14:5,
pages 455-468.

Wang, G., Li, X. and Wang, Z. (2016) APD3: the antimicrobial peptide
database as a tool for research and education. *Nucleic Acids Research*,
44 D1087-D1093
